.github/workflows/actions.yml

```diff
@@ -0,0 +1,63 @@
+# 59 23 * * *
+
+name: Update natural gas data
+
+on:
+  # Schedule to run on the 1st day of each month
+  schedule:
+    - cron: '59 23 * * *'
```

Comment:
@gradedSystem I think it should run on daily basis but the comment above is confusing. Also, why wouldn't we run at 00:00 utc?